src/utils/jwt.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const secretKey = process.env.JWT_KEY;
+/**
+ *
+ */
+export default class jwtHelper {
+  /**
+   * @param {object} payload - The details to be signed
+   * @param {string} secret - The JWT secret key
+   * @returns {string} The JWT signed token
+   */
+  static async generateToken(payload, secret = secretKey) {
```

Comment:
payload is not defined

Reply:
payload is the data you pass into the function
the data will be the user data